### ISSUE TITLE
Grab bag of small issues

### ISSUE THREAD

**Issue by robinwhittleton** [dajare/anthony-trollope_cousin-henry#1]
`content.opf` longdesc issues:
- neice -> niece
- `<abbr>` for Mr.
- `<em>` -> `<i>`
- I would say that inner-life reads better as inner life

General:
- Letter recipients with `<span epub:type="z3998:recipient">` semantics.

**Comment by dajare** [comment]
Thanks for the swift review, @robinwhittleton  - All fixed now, I believe.

Just a note on the last one:

> - Letter recipients with `<span epub:type="z3998:recipient">` semantics.

I did ponder that one, but was guided by the instructions at [SEMoS § 7.7.2.7][1]: 

> The name of the recipient of the letter, **when set out other than within a saluation** \[sic\] (for example a letter headed “To: John Smith Esquire”),...

And there are no such lines in the four *Cousin Henry* letters. Still, on checking the corpus, `recipient` is very frequently added **within** a "salutation". Is this a(nother) case of where practice and Manual are slightly out of step? (The [examples provided][2] also lack `recipient` semantics.)

I have, however, added them here. :)

If you're content with my corrections, it's ready for Alex, I suppose? Thanks again!

[1]: https://standardebooks.org/manual/1.7.0/single-page#7.7.2.7
[2]: https://standardebooks.org/manual/1.7.0/single-page#id4

**Comment by robinwhittleton** [comment]
The `Mr.` needs the tags escaped, but the rest looks good.

Re `recipient`, I wasn’t actually aware of that rule 😅 Let’s go with SEMOS on it.

**Comment by dajare** [comment]
> The `Mr.` needs the tags escaped, but the rest looks good.

Arghhh! Don't know why I've done so poorly on the long description this time. Will be more alert on the next one! (And, FIXED!)

> Re `recipient`, I wasn’t actually aware of that rule sweat_smile I don’t think it hurts to add them, so let’s stick with what you’ve got.

Oopsie: I saw this come in from email, and simply made the change to follow SEMOS, so I reverted ... but I see now you edited to "stick with what you've got". I can add the semantics back, of course! Let me know which you'd prefer; I'm happy to follow your lead.

Sorry for the aggro!
